lib/modules/platform/github/index.ts

```diff
@@ -759,26 +762,57 @@ export async function findPr({
 
 const REOPEN_THRESHOLD_MILLIS = 1000 * 60 * 60 * 24 * 7;
 
+async function remoteBranchExists(
```

Comment:
can we move this to a separate file because this file is already really big. then this function would be easier to test.